src/behavior/src/A32NX_Interior_FCU.xml

```diff
@@ -317,15 +317,23 @@
         <DefaultTemplateParameters>
             <NODE_ID>PUSH_AUTOPILOT_AUTOLAND_#SIDE#</NODE_ID>
             <SEQ_POWERED>1</SEQ_POWERED>
-            <NO_SEQ2 />
+            <NO_SEQ1/>
         </DefaultTemplateParameters>
 
         <UseTemplate Name="FBW_Push_Held">
-            <SEQ1_CODE>(L:A32NX_AUTOPILOT_AUTOLAND_WARNING, bool)</SEQ1_CODE>
-            <SEQ1_EMISSIVE_DRIVES_VISIBILITY>True</SEQ1_EMISSIVE_DRIVES_VISIBILITY>
+            <NO_SEQ1/>
+            <SEQ2_CODE>(L:A32NX_AUTOPILOT_AUTOLAND_WARNING, bool) (E:ABSOLUTE TIME, seconds) 1 % 0.4 &gt; and</SEQ2_CODE>
+            <SEQ2_EMISSIVE_DRIVES_VISIBILITY>True</SEQ2_EMISSIVE_DRIVES_VISIBILITY>
```

Comment:
Usage of this template is as follows:
```xml
            <UseTemplate Name="FBW_Airbus_Autoland_Warning">
                <SIDE>L</SIDE>
                <!-- Note there is only a single emissive. Once two emissives are available SEQ1 should use AC ESS SHED, and SEQ 2 should use AC 2. -->
                <SEQ_POWERED>(L:A32NX_ELEC_AC_ESS_SHED_BUS_IS_POWERED, Bool) (L:A32NX_ELEC_AC_2_BUS_IS_POWERED, Bool) or</SEQ_POWERED>
            </UseTemplate>
            <UseTemplate Name="FBW_Airbus_Autoland_Warning">
                <SIDE>R</SIDE>
                <!-- Note there is only a single emissive. Once two emissives are available SEQ1 should use AC ESS SHED, and SEQ 2 should use AC 2. -->
                <SEQ_POWERED>(L:A32NX_ELEC_AC_ESS_SHED_BUS_IS_POWERED, Bool) (L:A32NX_ELEC_AC_2_BUS_IS_POWERED, Bool) or</SEQ_POWERED>
            </UseTemplate>
```
Previously @komp1821 indicated there were two lights. Now it seems there's only one light. I think it's best to ask @komp1821 which bus(es) actually power that light, to ensure correctness.

Reply:
Hi! Seems like they are powered by both? Makes sense from redundant perspective. 

![bus](https://user-images.githubusercontent.com/433429/121821708-e4c3d000-cc9a-11eb-9036-a228be4712e9.png)